### ISSUE TITLE
Replace original Home Connect integration in Home Assistant core

### ISSUE THREAD

**Issue by stickpin** [ekutner/home-connect-hass#174]
Hi @ekutner,

Are you considering pushing your version to the Home Assistant core instead of the "original" one?
The original is not maintained as far as I can see.
Your version is excellent and provides all the missing functionality.
I guess users will support you as well.

Best Regards and Thanks a lot for the great integration!

**Comment by ChristophCaina** [comment]
see this discussion :)
https://github.com/ekutner/home-connect-hass/discussions/96

**Comment by ekutner** [comment]
I'm not sure what would be the benefit of doing that. 
The only pro I can think of is that will improve discoverability of this integration but it is already pretty easy to find, coming up as the second Google result when searching for "home assistant home connect". 
On the other hand doing so will reduce the update frequency to, at best, match the monthly HA release cycle. Even worse, it will not allow me to release beta versions of the integration to test specific improvements and fixes. This is especially important because Home Connect has a lot of inconsistencies and the API documentation is not complete, so it is not possible for me to test it well enough without getting feedback from people who have different appliances than I do. Without a proper beta process it is almost certain there will be bad releases that break the integration for some people.

Bottom line is that I see more negative than positive in making it the official integration, and that is before even considering the fact that it will break existing implementations and it's not even clear HA maintainers will agree to that. I'm open to counter arguments but as of now I don't plan on pursuing this path.

**Comment by stickpin** [comment]
Hi @ekutner,

Thanks for the detailed answer.
Except for the obvious benefit of people getting exposed to the best Home Connect integration, I have to admit, you are absolutely right. :)
Not all the integrations in HA get equal attention from the code owners and some PRs are stuck for weeks in the review.
I have a couple of ideas on how to improve a few things in your integration even further, but I want to try it myself first, before submitting the PR.

Thanks once again! :)
